cla-backend-go/v2/cla_manager/service.go

```diff
@@ -518,6 +520,13 @@ func (s *service) CreateCLAManagerRequest(contactAdmin bool, companyID string, p
 		return nil, ErrNotInOrg
 	}
 
+	// Check if user has organization
+	if lfxUser.Account.Name == "" {
+		msg := fmt.Sprintf("ACS service - 400 Bad Request - User: %s is not associated with any organization", userEmail)
```

Comment:
Even though this is a potential ACS issue, we want to report the error as 'EasyCLA Service' so the caller knows where the error originated.

Reply:
> Even though this is a potential ACS issue, we want to report the error as 'EasyCLA Service' so the caller knows where the error originated.

👍